regression/checkpr.sh

```diff
@@ -96,12 +96,15 @@ case $project in
     print_use; exit 1 ;;
 esac
 
-
-if ! [[ $LOGNAME == "kellyt" ]]; then
-  echo ""; echo "FATAL ERROR: Please use ccscs6 for manual use of checkpr.sh."
-  exit 1
-fi
-
+# Restrict the use of ccscs7.
+case $target in
```

Comment:
Possibly - there is a ticket to move some of the CI work load to another machine (maybe ccscs6).